services.yaml

```diff
@@ -4316,6 +4232,126 @@ paths:
           $ref: "#/responses/NotFoundError"
         401:
           $ref: "#/responses/UnauthorizedError"
+# Agents and Collectors Management
+  /services/{env_id}/overops-status:
+    get:
+      tags:
+        - Environment Management
+      summary: "List agents and collectors"
+      description: "Fetch OverOps live processes report"
+      parameters:
+        - in: path
+          name: env_id
+          description: Environment to use
+          type: string
+          required: true
+        - in: query
+          name: connected
+          description: Retrieve only processes with a connected OverOps collector/agent
+          type: string
+      responses:
+        200:
+          description: List of OverOps components status
+          schema:
+            type: object
+            properties:
+              agents:
+                type: array
+                items:
+                  $ref: "#/definitions/Agent"
+              collectors:
+                type: array
+                items:
+                  $ref: "#/definitions/Collector"
+        401:
+            $ref: "#/responses/UnauthorizedError"
+  /services/{env_id}/overops-status/agents:
+    get:
+      tags:
+        - Environment Management
+      summary: "List agents"
+      description: "Fetch agents live processes report"
+      parameters:
+        - in: path
+          name: env_id
+          description: Environment to use
+          type: string
+          required: true
+        - in: query
+          name: connected
+          description: Retrieve only processes with an attached agents
+          type: string
+      responses:
+        200:
+          description: List of agents
+          schema:
+            properties:
+              agents:
+                type: array
+                items:
+                  $ref: "#/definitions/Agent"
+        401:
+            $ref: "#/responses/UnauthorizedError"
+  /services/{env_id}/overops-status/collectors:
+    get:
+      tags:
+        - Environment Management
+      summary: "List collectors"
+      description: "Fetch collectors live processes report"
+      parameters:
+        - in: path
+          name: env_id
+          description: Environment to use
+          type: string
+          required: true
+        - in: query
+          name: connected
+          description: Retrieve only processes of a connected OverOps collector
+          type: string
+      responses:
+        200:
+          description: List of Collectors
+          schema:
+            properties:
+              collectors:
+                type: array
+                items:
+                  $ref: "#/definitions/Collector"
+        401:
+            $ref: "#/responses/UnauthorizedError"
+  /services/{env_id}/agents:
```

Comment:
didn't we agree that this will be under the same overops-status? otherwise it's weird that there's one endpoint for POST and one for GET...